sessions/forecasting-models.qmd

```diff
@@ -199,18 +199,19 @@ $$
 ::: {.callout-note, collapse = TRUE}
 ## What behaviour would we expect from this model?
 
+Again we look at an R function that performs this random walk
```

Comment:
Autoregressive not ransom walk. Can be fixed in another pr

Reply:
have rephrased